content/en/platform/corda/4.8/enterprise/cordapps/cordapp-build-systems.md

```diff
@@ -319,7 +319,7 @@ You could sign the CorDapp automatically by:
 
 ### Run development and production modes
 Nodes only accept CorDapps signed by Corda development certificates when running in development mode. If you need to run a CorDapp signed by the (default) development key in the production mode (for example, for testing), add the `cordappSignerKeyFingerprintBlacklist = []` property set to an empty list. See
-[Configuring a node](../node/setup/corda-configuration-file.md#corda-configuration-file-signer-blacklist)).
+[Configuring a node](../../../../../../en/platform/corda/4.8/enterprise/node/setup/corda-configuration-file.html#corda-configuration-file-signer-blacklist)).
```

Comment:
I can't find this header on that page. Closest I got was #limitations.

Reply:
Yeah this was problematic in another link PR too. I don't know if there was originally more to this section or another section entirely, but #limitations does seem to be the best fit.